receiver/postgresqlreceiver/client.go

```diff
@@ -458,9 +458,9 @@ func (c *postgreSQLClient) getReplicationStats(ctx context.Context) ([]replicati
 	query := `SELECT
 	client_addr,
 	coalesce(pg_wal_lsn_diff(pg_current_wal_lsn(), replay_lsn), -1) AS replication_bytes_pending,
-	coalesce(write_lag, -1),
-	coalesce(flush_lag, -1),
-	coalesce(replay_lag, -1)
+	extract('epoch' from coalesce(write_lag, '-1 seconds')),
```

Comment:
Is this query validated as part of the tests?

Reply:
No, it is not validated (which is why it was broken previously and we weren't aware). As we don't have multiple hosts & replication set up in integration tests, the query can't be validated.